e2e/cypress/plugins/index.js

```diff
@@ -18,4 +18,6 @@
 module.exports = (on, config) => {
   // `on` is used to hook into various events Cypress emits
   // `config` is the resolved Cypress config
+  config.ignoreTestFiles = "**/z_examples/*.spec.js";
```

Comment:
Great idea! Can we just leave it examples and not do the kludgey rename?

Reply:
Could we just do this?

```suggestion
  config.ignoreTestFiles = "**/examples/*.spec.js";
```